tests/security/libserf/libserf.pm

```diff
@@ -0,0 +1,125 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Setup Apache2 with SSL enabled and test libserf by using SVN.
+# Maintainer: Starry Wang <starry.wang@suse.com> Ben Chou <bchou@suse.com>
+# Tags: poo#110434, tc#1769948
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Architectures;
+
+sub run {
+    my ($self) = @_;
+
+    select_console('root-console');
+
+    # Setup Apache2 server with SSL enabled
+    zypper_call('in apache2');
+    # Activate the SSL Module
+    assert_script_run('a2enmod ssl');
+    assert_script_run('echo "127.0.0.1 example-ssl.com" > /etc/hosts');
+    # Prepare certificates
+    assert_script_run('gensslcert -n example-ssl.com -e webmaster@example.com');
+    my $vhost_ssl_conf_path = '/etc/apache2/vhosts.d/vhost-ssl.conf';
+    type_string("cat >> $vhost_ssl_conf_path << EOF
+<IfDefine SSL>
+<IfDefine !NOSSL>
+<VirtualHost _default_:443>
+
+    DocumentRoot \"/srv/www/vhosts/example-ssl.com\"
+    ServerName example-ssl.com
+    ServerAdmin webmaster\@example-ssl.com
+    ErrorLog /var/log/apache2/example-ssl.com-error_log
+    TransferLog /var/log/apache2/example-ssl.com-access_log
+    CustomLog /var/log/apache2/example-ssl_request_log   ssl_combined
+
+    SSLEngine on
+    SSLCertificateFile /etc/apache2/ssl.crt/example-ssl.com-server.crt
+    SSLCertificateKeyFile /etc/apache2/ssl.key/example-ssl.com-server.key
+    SSLCertificateChainFile /etc/apache2/ssl.crt/example-ssl.com-ca.crt
+
+    <Directory \"/srv/www/vhosts/example-ssl.com\">
+    Options Indexes FollowSymLinks
+    AllowOverride None
+    <IfModule !mod_access_compat.c>
+        Require all granted
+    </IfModule>
+    <IfModule mod_access_compat.c>
+        Order allow,deny
+        Allow from all
+    </IfModule>
+    </Directory>
+</VirtualHost>
+
+</IfDefine>
+</IfDefine>
+EOF
+");
+    assert_script_run("cat $vhost_ssl_conf_path | grep SSLCertificate");
+    assert_script_run("sed -i \"/^APACHE_SERVER_FLAGS=*/c\\APACHE_SERVER_FLAGS=\\\"SSL\\\"\" /etc/sysconfig/apache2");
+    assert_script_run('cat /etc/sysconfig/apache2 | grep APACHE_SERVER_FLAGS=');
+    save_screenshot;
+
+    # Setup subversion configration for apache
+    zypper_call('in subversion-server');
```

Comment:
can you please install all required pkgs in one command?

Reply:
Done, thanks for the comments.